generic-database-plugin/src/test/java/co/cask/db/batch/action/DBActionTestRun.java

```diff
@@ -38,6 +39,7 @@
 /**
  * Test for DBAction Plugin
  */
+@Ignore // Probably need to be converted to specific database provider and moved in integration tests repo.
```

Comment:
why ignore? The generic DB plugins should still work and be used with hypersql to test the plugins

Reply:
I temporarily ignored this test before we made decision to move generic plugin to a separate module. We'll use existing hsql tests to verify generic-database-plugin functionality and write dedicated tests for DB-specific plugins.

Reply:
This PR should just be adding a mysql specific plugin. It should *not* break the existing DB plugins.

Reply:
Since I changed the database-plugin classes to abstract, and also removed the "columns" field from generic connection config class, some tests demanded minor changes to work, and I submitted them in last commit